docs/src/components/_overview.md

```diff
@@ -30,4 +30,3 @@ The rest of this section provides a high-level breakdown of the components that
 - [Libraries and Functions](./libraries_and_functions.md)
 - [Programs and Authorizations](./programs_and_authorizations.md)
 - [Middleware](./middleware.md)
-- [Storage Accounts](./storage_account.md)
```

Comment:
Is this deletion on purpose?

Reply:
Yes. When clicked, takes me to a page that says "Document not found (404)". didn't see a section for "Storage Accounts" in the side bar to relink it to so propose deleting it (at least until there's a page to link it to)